libs/core/src/components.d.ts

```diff
@@ -191,6 +191,53 @@ export namespace Components {
          */
         "variant": 'inline' | 'plain';
     }
+    interface SageSwitch {
+        /**
+          * Sets state of input control
+         */
+        "checked": boolean;
```

Comment:
Just a heads up that prop descriptions are slated for discussion during the consistency meeting. May be worth updating this or checkbox post discussion.